coral-component-masonry/src/scripts/MasonryColumnLayout.js

```diff
@@ -279,6 +279,32 @@ class MasonryColumnLayout extends MasonryLayout {
       nextItem.focus();
       // prevent scrolling at the same time
       event.preventDefault();
+    } 
+    else {
+      // in case there is no item in the same column, we should move to first item in next column for down
+      // and last item of previous column for up key
+      let columnIndex = currentLayoutData.columnIndex;
+      if (up) {
+        if (columnIndex > 0) {
+          // move to last item of previous column
+          let prevColumn = this._columns[columnIndex - 1];
+          if (prevColumn) {
+            nextItem = prevColumn.items[prevColumn.items.length - 1]; // last item of previous column
```

Comment:
In order to overwrite `nextItem`, we have to use `let` instead of `const` at [line 276](https://github.com/adobe/coral-spectrum/pull/79/files#diff-e292e1bd4d551eb4bdb4769f46242dd5R276).